src/experiments.js

```diff
@@ -189,9 +194,10 @@ export function experimentTogglesOrNull(win) {
  * @param {!Window} win
  * @return {!Object<string, boolean>}
  */
-function getExperimentTogglesFromCookie(win) {
-  const experimentCookie = getCookie(win, COOKIE_NAME);
-  const tokens = experimentCookie ? experimentCookie.split(/\s*,\s*/g) : [];
+function getExperimentToggles(win) {
+  const experimentsString =
+    'localStorage' in win ? win.localStorage.getItem(LOCAL_STORAGE_KEY) : '';
```

Comment:
This probably needs to be try-block guarded.

Reply:
Done, thanks.